pyproject.toml

```diff
@@ -45,6 +45,8 @@
     name = "Contributor-facing changes"
     showcontent = true
 
+[build-system]
+requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]
```

Comment:
`wheel` shouldn't be in this list, `setuptools`' PEP 517 build backend auto-adds it already.

Reply:
(and of course move this to a separate PR as well)

Reply:
See also: https://github.com/ansible/ansible-navigator/issues/554